Frontend/android-native/mobility-driver/src/main/java/in/juspay/mobility/driver/MobilityDriverBridge.java

```diff
@@ -622,6 +631,27 @@ public int methodArgumentCount(String functionName) {
     
     @Override
     public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
+        switch (requestCode) {
+
+            case HV_REQUEST_CODE:
+                if (resultCode == RESULT_OK) {
+                    HyperKycResult result = data.getParcelableExtra("hyperKycResult");
+                    switch (result.getStatus()) {
+                        case "USER_CANCELLED":
+                            return true;
```

Comment:
Why return true ?

Also can you please confirm if we need to do API call from frontend then
Should it be in native or ps
If ps then add callbacks (we wil sit together and do)